src/cc/tests/CMakeLists.txt

```diff
@@ -58,7 +58,6 @@ foreach(test_dir ${test_dirs})
 endforeach()
 
 set(test_binary test.t)
-message(STATUS "Test sources: ${test_sources}")
```

Comment:
Yes, actually, there is a lot of benefit to having one pull request per component. I should start doing that. I'm sorry. I'll update this pull request to reflect that and open a few more as well.